jiant/proj/main/export_model.py

```diff
@@ -29,8 +29,12 @@ def export_model(model_type, output_base_path, model_class, tokenizer_class):
     os.makedirs(tokenizer_fol_path, exist_ok=True)
     os.makedirs(model_fol_path, exist_ok=True)
 
-    model_path = os.path.join(model_fol_path, f"{model_type}.p")
-    model_config_path = os.path.join(model_fol_path, f"{model_type}.json")
+    # nyu_mll/roberta-base creates spurious directory
+    model_filename = model_type.replace("/", "-")
```

Comment:
@zphang the model name `nyu_mll/roberta-base` contains `/` which causes a new folder to be created. Do any alternatives come to mind to handle this situation?

Reply:
Yep. This was resolved with the update from: https://github.com/jiant-dev/jiant/pull/129

How about I break that off as a standalone PR?

Reply:
@zphang That would be great! @phu-pmh Let's adapt this PR based on those changes.